pkg/apis/configuration/validation/validation.go

```diff
@@ -796,3 +797,21 @@ func validateVirtualServerRouteSubroutes(routes []v1alpha1.Route, fieldPath *fie
 
 	return allErrs
 }
+
+func rejectPlusResourcesInOSS(upstream v1alpha1.Upstream, idxPath *field.Path, isPlus bool) field.ErrorList {
```

Comment:
I think we can create unit tests for this method as all the other validation methods have. What do you think?